tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Prepare a Relay Chain
+description: This tutorial will guide you through preparing a relay chain so that you can connect a test parachain node to it for local testing.
+---
+
+# Prepare Relay Chain
+
+## Introduction
+
+This tutorial illustrates how to configure and spin up a local relay chain. The local relay chain is required to set up a local testing environment to which a test parachain node can connect. Setting up a local relay chain is a crucial step in parachain development. It allows developers to test their parachains in a controlled environment, simulating the interaction between a parachain and the relay chain without needing a live network. This local setup facilitates faster development cycles and easier debugging.
+
+The scope of this tutorial includes:
+
+- Installing necessary components for a local relay chain
+- Configuring the relay chain settings
+- Starting and running the local relay chain
+- Verifying the relay chain is operational
+
+## Prerequisites
+
+Before diving into this tutorial, it's recommended that you have a basic understanding of how adding trusted nodes works in Polkadot. For further information about this process, refer to the [Add Trusted Nodes](TODO: add the absolute path to the Add Trusted Nodes tutorial) tutorial.
+
+To complete this tutorial, ensure that you have:
+
+- Configured your environment for Substrate development by installing [Rust and the Rust toolchain](https://docs.substrate.io/install/){target=\_blank}
+- Completed [Build a local blockchain](TODO: the add absolute path to the Build a local blockchain tutorial){target=\_blank} tutorial and know how to compile and run a Substrate node
+
+## Build a Local Relay Chain
+
+To build a local relay chain, follow these steps:
+
+1. Clone the most recent release branch of the Polkadot SDK repository to prepare a stable working environment
+
+    ```bash
+    git clone --depth 1 --branch polkadot-stable2407-2 https://github.com/paritytech/polkadot-sdk.git
+    ```
+
+    !!! note
+        The branch `polkadot-stable2407-2` is used in this tutorial since it is the branch that contains the latest stable release of the Polkadot SDK. You can find the latest release of the Polkadot SDK on the [Release](https://github.com/paritytech/polkadot/releases){target=\_blank} tab on the Polkadot GitHub repository.
```

Comment:
The link is wrong. That repository is archived. Did you mean `https://github.com/paritytech/polkadot-sdk/releases`?